cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -396,6 +396,18 @@ def set_sentry_client(self, sentry_url=None):
     def get_current_ip(self):
         return socket.gethostbyname(socket.gethostname())
 
+    def set_scws_realted_params(self):
+        # it is nowhere necessary within the dispatcher-app,
+        # but it is re-attached to the url within the email
+        # when sending it since it is used by the frontend
+        self.use_scws = self.par_dic.pop('use_scws', None)
+        #
+        if 'scw_list' in self.par_dic.keys():
```

Comment:
I think if `scw_list` is in params, and the query is from api, the eventual URL to be parsed by the frontend also has to contain `use_scws`. Else url will not work in the frontend.

Could you verify this?

Reply:
Ah true! It makes sense. 

So it should be added anyway when creating the url for the email.

Reply:
And the value that it should have is `form_list`, so that the right radio button will be selected.